modules/client-configuration/pages/supported-features-ubuntu.adoc

```diff
@@ -18,183 +18,224 @@ The icons in this table indicate:
 
 ifeval::[{suma-content} == true]
 
-[cols="1,1,1", options="header"]
+[cols="1,1,1,1", options="header"]
 .Supported Features on {ubuntu} Operating Systems
 |===
 
 | Feature
 | {ubuntu}{nbsp}20.04
 | {ubuntu}{nbsp}22.04
+| {ubuntu}{nbsp}24.04
 
 | Client
 | {check}
 | {check}
+| {check}
 
 | System packages
 | {ubuntu} Community
 | {ubuntu} Community
+| {ubuntu} Community
 
 | Registration
 | {check}
 | {check}
+| {check}
 
 | Install packages
 | {check}
 | {check}
+| {check}
 
 | Apply patches
 | {check}
 | {check}
+| {check}
 
 | Remote commands
 | {check}
 | {check}
+| {check}
 
 | System package states
 | {check}
 | {check}
+| {check}
 
 | System custom states
 | {check}
 | {check}
+| {check}
 
 | Group custom states
 | {check}
 | {check}
+| {check}
 
 | Organization custom states
 | {check}
 | {check}
+| {check}
 
 | System set manager (SSM)
 | {check}
 | {check}
+| {check}
 
 | Product migration
 | N/A
 | N/A
+| N/A
 
 | Basic Virtual Guest Management {star}
 | {check}
 | {check}
+| {check}
 
 | Advanced Virtual Guest Management {star}
 | {check}
 | {check}
+| {check}
 
 | Virtual Guest Installation (Kickstart), as Host OS
 | {cross}
 | {cross}
+| {cross}
 
 | Virtual Guest Installation (image template), as Host OS
 | {check}
 | {check}
+| {check}
 
 | System deployment (PXE/Kickstart)
 | {cross}
 | {cross}
+| {cross}
 
 | System redeployment (Kickstart)
 | {cross}
 | {cross}
+| {cross}
 
 | Contact methods
 | {check} ZeroMQ, Salt-SSH
 | {check} ZeroMQ, Salt-SSH
+| {check} ZeroMQ, Salt-SSH
 
 | Works with {productname} Proxy
 | {check}
 | {check}
+| {check}
 
 | Action chains
 | {check}
 | {check}
+| {check}
 
 | Staging (pre-download of packages)
 | {check}
 | {check}
+| {check}
 
 | Duplicate package reporting
 | {check}
 | {check}
+| {check}
 
 | CVE auditing
 | {question}
 | {question}
+| {question}
 
 | SCAP auditing
 | {question}
 | {question}
+| {question}
 
 | Package verification
 | {cross}
 | {cross}
+| {cross}
 
 | Package locking
 | {check}
 | {check}
+| {check}
 
 | Maintenance Windows
 | {check}
 | {check}
+| {check}
 
 | System locking
 | {cross}
 | {cross}
+| {cross}
 
 | System snapshot
 | {cross}
 | {cross}
+| {cross}
 
 | Configuration file management
 | {check}
 | {check}
+| {check}
 
 | Package profiles
 | {check} Profiles supported, Sync not supported
 | {check} Profiles supported, Sync not supported
+| {check} Profiles supported, Sync not supported
 
 | Power management
 | {check}
 | {check}
+| {check}
 
 | Monitoring server
 | {cross}
 | {cross}
+| {cross}
 
 | Monitored clients
 | {check}
 | {check}
+| {check}
 
 | Docker buildhost
 | {question}
 | {question}
+| {question}
 
 | Build Docker image with OS
 | {check}
 | {check}
+| {check}
 
 | Kiwi buildhost
 | {cross}
 | {cross}
+| {cross}
 
 | Build Kiwi image with OS
 | {cross}
 | {cross}
+| {cross}
 
 | Recurring Actions
 | {check}
 | {check}
-| {salt}
-| {salt}
+| {check}
```

Comment:
There's a conflict here right now, which should disappear when you rebase on top of the latest master, if I'm not mistaken. These `| {salt}` lines are not there any more right now.